js/customize-snapshots.js

```diff
@@ -265,6 +285,214 @@
+			if ( _.isUndefined( control ) || _.isUndefined( control.notifications ) ) {
+				return;
+			}
+
+			if ( snapshot.conflict.currentRequest ) {
+				snapshot.conflict.currentRequest.abort();
+				snapshot.conflict.currentRequest = null;
+			}
+
+			if ( snapshot.conflict.conflictRequestTimeout ) {
+				clearTimeout( snapshot.conflict.conflictRequestTimeout );
+				snapshot.conflict.conflictRequestTimeout = null;
+			}
+
+			if ( ! _.isFunction( setting.findControls ) ) {
+				return;
+			}
+
+			snapshot.conflict.controlsWithPendingRequest[ setting.id ] = setting.findControls();
+			snapshot.conflict.conflictRequestTimeout = _.delay( snapshot.sendConflictRequest, snapshot.conflict.refreshBuffer );
+		},
+
+		/**
+		 * Send Conflict Request
+		 *
+		 * @return {void}
+		 */
+		sendConflictRequest: function() {
+			var snapshot = this, settingIds;
+
+			settingIds = _.keys( snapshot.conflict.controlsWithPendingRequest );
+
+			if ( _.isEmpty( settingIds ) ) {
+				return;
+			}
+
+			snapshot.conflict.currentRequest = wp.ajax.post( 'customize_snapshot_conflict_check', {
+				setting_ids: settingIds,
+				nonce: snapshot.data.conflictNonce,
+				changeset_uuid: api.settings.changeset.uuid
+			} );
+
+			snapshot.conflict.currentRequest.done( function( response ) {
+				var multiple, controls, buttonTemplate, notification, notificationsContainer;
+
+				if ( ! _.isEmpty( response ) ) {
+					_.each( response, function( value, key ) {
+
+						snapshot.conflict.controlThickboxes[ key ] = $( $.trim( snapshot.conflict.warningTemplate( {
+							setting_id: key,
+							conflicts: value
+						} ) ) );
+
+						multiple = false;
+						controls = snapshot.conflict.controlsWithPendingRequest[ key ];
+						buttonTemplate = $( $.trim( snapshot.conflict.conflictButtonTemplate( {
+							setting_id: key
+						} ) ) );
+
+						_.each( controls, function( control ) {
+							control.notificationsTemplate = snapshot.conflict.notificationTemplate;
+							notification = new api.Notification( snapshot.conflict.notificationCode, {
+								type: 'warning',
+								message: $.trim( buttonTemplate.html() )
+							} );
+
+							control.notifications.add( snapshot.conflict.notificationCode, notification );
+
+							if ( ! multiple ) {
+								notificationsContainer = control.container.find( '.customize-control-notifications-container' );
+								if ( notificationsContainer.length ) {
+									snapshot.conflict.controlThickboxes[ key ].insertAfter( notificationsContainer );
+									snapshot.updateConflictValueMarkup( key, snapshot.conflict.controlThickboxes[key] );
+								}
+							}
+
+							control.container.find( '.snapshot-conflicts-button' ).show();
+							multiple = true;
+						} );
+					} );
+				}
+				snapshot.conflict.controlsWithPendingRequest = {};
+			} );
+		},
+
+		/**
+		 * Update markup with current value
+		 *
+		 * @param {string} settingId setting id.
+		 * @param {object} selector control container or conflict markup container.
```

Comment:
jsdoc should have the description starting with capital. A hyphen can separate the param name from description:

```
@param {jQuery} selector - Control container or conflict markup container.
```